django/db/models/base.py

```diff
@@ -324,7 +326,7 @@ def _prepare(cls):
 
             # defer creating accessors on the foreign class until we are
             # certain it has been created
-            def make_foreign_order_accessors(field, model, cls):
+            def make_foreign_order_accessors(cls, model, field):
```

Comment:
I'd leave these argument order changes out, unless there's a really good reason to change them.